src/index.ts

```diff
@@ -435,10 +441,8 @@ export const plugin = (opts?: { debug: boolean }): Plugin => {
           const message = `${viteProjectPath(
             id,
           )}: NO MAIN .elm file is requested to transform by vite. Probably, this file is just a depending module`
-          console.error(message)
           throw message
         } else {
-          console.error(e)
```

Comment:
I removed these so we don't get double errors in the CLI. Was there a specific reason to do `console.error`?

Reply:
unnecessary at all! thanks for pointing this out